tests/utils_tests/test_autoreload.py

```diff
@@ -185,3 +198,59 @@ def test_check_errors_catches_all_exceptions(self):
             with self.assertRaises(Exception):
                 autoreload.check_errors(import_module)('test_exception')
         self.assertFileFound(filename)
+
+
+class CleanFilesTestCase(SimpleTestCase):
```

Comment:
`CleanFilesTests` (TestCase implies the class doesn't contain any tests and is meant to be used a test base).